clearwater-etcd/usr/share/clearwater/bin/poll_etcd_cluster.sh

```diff
@@ -23,6 +23,18 @@ ALARM_TO_RAISE_FILE="/tmp/.clearwater_etcd_alarm_to_raise"
 
 . /etc/clearwater/config
 
+
+
+log_bad_poll()
+{
+  # We log the bad poll and truncate if necessary to keep the file size bounded
+  echo $(date) >> "/var/log/clearwater-etcd/etcd_cluster_bad_polls.txt"
```

Comment:
It feels a bit weird that you change the actual file twice. Why not always modify the .tmp file, then copy that across once at the end?

Reply:
I think it logically makes sense to write the logs and then truncate the file as necessary. If I were to change the .tmp file only, I would first have to copy the original, then do the modifications, tail last 1000 lines and finally delete the .tmp file, which takes more steps. When I searched for how to tail a log file, this seems to be the standard way to do it.